homeassistant/util/location.py

```diff
@@ -17,16 +17,35 @@
      'city', 'zip_code', 'time_zone', 'latitude', 'longitude',
      'use_fahrenheit'])
 
+DATA_SOURCE = ['https://freegeoip.io/json/', 'http://ip-api.com/json']
```

Comment:
Hey guys, I set up freegeoip.io as a mirror to freegeoip.net. I'm using it in production myself. It is beeing actively monitored so it should stay up.

Should you notice any issues, let me know!